src/services/services.ts

```diff
@@ -2101,7 +2101,7 @@ namespace ts {
         };
     }
 
-    export function preProcessFile(sourceText: string, readImportFiles = true): PreProcessedFileInfo {
+    export function preProcessFile(sourceText: string, readImportFiles = true, isJavaScript = false): PreProcessedFileInfo {
         let referencedFiles: FileReference[] = [];

```

Comment:
since the other parameter is primitive, i would name this "readRequire" or "includeRequireCalls"